back/app/models/home_page.rb

```diff
@@ -15,20 +16,86 @@
 #  banner_avatars_enabled                   :boolean          default(TRUE), not null
 #  banner_enabled                           :boolean          default(TRUE), not null
 #  banner_layout                            :string           default("full_width_banner_layout"), not null
-#  banner_signed_in_header                  :jsonb            not null
-#  banner_signed_in_text                    :jsonb            not null
+#  banner_signed_in_header_multiloc         :jsonb            not null
+#  banner_signed_in_text_multiloc           :jsonb            not null
 #  banner_signed_in_type                    :string           default("no_button"), not null
 #  banner_signed_in_url                     :string
-#  banner_signed_out_header                 :jsonb            not null
-#  banner_signed_out_subheader              :jsonb            not null
+#  banner_signed_out_header_multiloc        :jsonb            not null
+#  banner_signed_out_subheader_multiloc     :jsonb            not null
 #  banner_signed_out_header_overlay_color   :string
 #  banner_signed_out_header_overlay_opacity :integer
-#  banner_signed_out_text                   :jsonb            not null
+#  banner_signed_out_text_multiloc          :jsonb            not null
 #  banner_signed_out_type                   :string           default("sign_up_button"), not null
```

Comment:
Why did you decide to remove `cta` from this and similar names? Probably, it's not 100% clear it's the type of CTA and not the type of the banner.